lntest/harness_assertion.go

```diff
@@ -377,6 +377,56 @@ func (h *HarnessTest) AssertChannelExists(hn *node.HarnessNode,
 	return channel
 }
 
+// AssertChannelActive checks if a channel identified by the specified channel
+// point is active.
+func (h *HarnessTest) AssertChannelActive(hn *node.HarnessNode,
+	cp *lnrpc.ChannelPoint) *lnrpc.Channel {
+
+	return h.AssertChannelStatus(hn, cp, true)
+}
+
+// AssertChannelInactive checks if a channel identified by the specified channel
+// point is inactive.
+func (h *HarnessTest) AssertChannelInactive(hn *node.HarnessNode,
+	cp *lnrpc.ChannelPoint) *lnrpc.Channel {
+
+	return h.AssertChannelStatus(hn, cp, false)
+}
+
+// AssertChannelStatus asserts that a channel identified by the specified
+// channel point exists from the point-of-view of the node and that it is either
+// active or inactive depending on the value of the active parameter.
+func (h *HarnessTest) AssertChannelStatus(hn *node.HarnessNode,
```

Comment:
let's make this private? Reason being it'd be easier for caller to finish the auto-complete in the editor.

Reply:
Do itests warrant an update of the release notes?

Reply:
Good question - atm no, but maybe we should think about it in future releases.